.azuredevops/deploy-platform-core.yaml

```diff
@@ -95,19 +95,29 @@ extends:
           - name: aks-cluster
             path: infra/managed-cluster
             resourceGroupName: $(aksResourceGroup)
+            postDeployScriptsList:
+              - displayName: Link the Private DNS Zone to Central networks
+                scriptPath: /scripts/ado/Trigger-LinkPrivateDNSZones.ps1
+                type: PowerShell
+                useSystemAccessToken: true
+                scriptArguments: >
+                  -privateDnsZoneName "$(dnsResourceNamePrefix)$(nc_resource_dnszone)$(nc_instance_regionid)01.privatelink.uksouth.azmk8s.io"
```

Comment:
where it has "ukSouth" on line 104 after privateline., we should add a region var?